frontend/src/metabase/admin/upsells/UpsellHosting.tsx

```diff
@@ -21,6 +21,7 @@ export const UpsellHosting = ({ source }: { source: string }) => {
       buttonLink="https://www.metabase.com/cloud"
       illustrationSrc={RocketGlobeIllustrationSrc}
       source={source}
+      maxWidth={202}
```

Comment:
This behaves the same as previously we set `max-height: 200px` and `box-sizing: content-box`

I need this change because `box-sizing: content-box` causes horizontal scrolling to appear when used in the embed modal and changing this to `box-sizing: border-box` wraps the title.
![Screenshot 2024-07-11 at 8 10 09 PM](https://github.com/metabase/metabase/assets/1937582/80e3479d-6db9-4b6b-adfc-52542dbaf795)

Reply:
makes sense to me, I'm not sure we need to care all that much about text wrapping, since it will look different in all locales.

Reply:
Yeah, I agree that we shouldn't worry too much, especially if it means that we have to use magic numbers.

On that note, could you use a constant (i.e. `UPSELL_BADGE_WIDTH = 202` or something) and make a comment about it? Otherwise someone else will remove it